projeto/src/components/Stopwatch/index.tsx

```diff
@@ -1,29 +1,19 @@
 import { TimerView } from '../TimerView/index'
-import { date } from '../../common/utils/date'
 import './style.scss'
 
 interface IStopwatchProps {
   time: number;
-  setTime: React.Dispatch<React.SetStateAction<number>>;
-  onFinish:() => void;
+  initiateCountdown:() => Promise<void>;
```

Comment:
No geral para cursos da Alura tentamos deixar o código o máximo em pt-BR possível. Estamos confirmando se o pessoal da ambev o que eles acham e se for tudo bem precisamos traduzir o código

Reply:
Com certeza! Na hora de alterar o repositório realmente farei tudo em inglês, fica muito mais fácil para os alunos